cfme/storage/volume.py

```diff
@@ -402,6 +408,10 @@ def status(self):
             self.refresh()
         else:
             view.browser.refresh()
+        try:
+            view.entities.get_entity(name=self.name, surf_pages=True)
+        except ItemNotFound:
+            return False
 
         for item in view.entities.elements.read():
```

Comment:
As you are using `get_entity`  now no need of this iteration over the table.
```
try:
    ent = view.entities.get_entity(name=self.name, surf_pages=True)
    return ent.data["status"]
except ItemNotFound:
    return False
```

Reply:
Nice, I tried to find where that entity has stored elements data...But haven't checked .data.
Thanks and reworked.